src/main/resources/liquibase/changeset/06.01.2022_1839_create_authors_table.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+  http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+    <changeSet author="aliaksandr" id="06.01.2022_1839_create_tags_table">
+        <createTable tableName="authors">
+            <column name="id" type="BIGINT" autoIncrement="true">
```

Comment:
Are you sure about autoIncrement
You use sequence in your code
Please fix it

Reply:
Ok